package/network/services/hostapd/files/hostapd.sh

```diff
@@ -408,7 +409,7 @@ hostapd_set_psk_file() {
 	json_get_vars mac vid key
 	set_default mac "00:00:00:00:00:00"
```

Comment:
With SAE, there can only be one default key because - unlike with PSK2 - the AP can't check a PMK on its own.
That means there can be only one definition for MAC `00:00:00:00:00:00`.
Maybe it makes sense to make that more explicit?

Reply:
(that is unless we start using `id`?)
hmm

Reply:
Password ID support implemented via https://github.com/openwrt/openwrt/pull/14448/commits/e05e0c35fd7c67c576a896c37476624a5e49f916
Exiting function with a message to stdout if a password is already set for default MAC with no ID.